### ISSUE TITLE
WPF BlazorWebView TabStopOrder not working

### ISSUE THREAD

**Issue by MichaelPeter** [dotnet/maui#2537]
### Describe the bug
I played arround with WPF and BlazorWebView, since I know Keyboard control is VERY importaint for powerusers I checked if the tab stop interaction works with the web view:

As you see the tab order should be First the Button, then the Button inside the BlazorWebView and then the Combo Box which drops down over the webside.

But the Website is always last, would it be possible to make the browser control to accept tab stop?

PS: Have not yet checked how to make keyboard shortcut interactions.

```
        <StackPanel Grid.Row="0" Orientation="Horizontal">
            <TextBlock VerticalAlignment="Center" Text="ComboBox: " />
            <ComboBox TabIndex="2" Margin="2,5,0,5" Width="80">
                <ComboBoxItem>Item 1</ComboBoxItem>
                <ComboBoxItem>Item 2</ComboBoxItem>
                <ComboBoxItem>Item 3</ComboBoxItem>
                <ComboBoxItem>Item 4</ComboBoxItem>
                <ComboBoxItem>Item 5</ComboBoxItem>
                <ComboBoxItem>Item 6</ComboBoxItem>
            </ComboBox>
            <TextBlock VerticalAlignment="Center" Margin="5,0,0,0" Text="Button Tab Index: " />
            <Button TabIndex="0"  Margin="2,5,0,5">ButtonTabIdx3</Button>
        </StackPanel>
        
        <blazor:BlazorWebView TabIndex="1" IsTabStop="True" Grid.Row="1" HostPage="wwwroot/index.html" Services="{StaticResource services}">
            <blazor:BlazorWebView.RootComponents>
                <blazor:RootComponent Selector="#app" ComponentType="{x:Type local:Counter}" />
            </blazor:BlazorWebView.RootComponents>
        </blazor:BlazorWebView>
```

### To Reproduce
In this sample https://github.com/MichaelPeter/BlazorDesktopWPFTestTabStop/blob/master/README.md
I made it reproducable

### Further technical details
- ASP.NET Core version
- Include the output of `dotnet --info`
- The IDE (VS / VS Code/ VS4Mac) you're running on, and its version

**Comment by Eilon** [comment]
Interesting. The WPF WebView2 control doesn't have a `TabIndex` property, but the BlazorWebView, which wraps it, does. When I try to use WebView2 directly and don't explicitly set TabIndex on any control, all tab order works as expected (I can tab between WPF controls and HTML controls as if it's all in the correct visual order). But since WebView2 doesn't have TabIndex, once I set TabIndex on all _other_ controls, the WebView2 ends up as _last_ in the tab order, but otherwise everything works as expected.

I need to see how we can get BlazorWebView to behave properly, at least as well as WebView2, but hopefully a bit better by supporting TabIndex.
